src/main/scala/io/iohk/praos/crypto/VerifiableRandomFunction.scala

```diff
@@ -2,33 +2,29 @@ package io.iohk.praos.crypto
 
 import akka.util.ByteString
 
-/**
-  * VrfProof contains a fresh random value generated using the user public key, a random seed and
-  * VerifiableRandomFunction.prove method. And also contains the proof, that will be used later,
-  * with the user public key, for verify that the random value was generated by him.
-  */
-case class VrfProof(random: RandomValue, proof: ByteString)
-
-object VrfProof {
-  // TODO: When the VrfProof be well defined, change to the true random length
-  val randomLength = 32
-}
 
 trait VerifiableRandomFunction {
 
-  def prove(privateKey: Key, randomSeed: RandomValue): VrfProof
+  type VrfProof = ByteString
```

Comment:
Minor detail: Maybe we should move this to `object VerifiableRandomFunction`?

Reply:
IMHO, the implementation of the `VrfProof` type should not depend on an specific implementation of the `VerifiableRandomFunction` trait.

Reply:
I misunderstood Nico's argument: I will move the type definition to a companion object.

Reply:
Resolved